### ISSUE TITLE
Writing User Row

### ISSUE THREAD

**Issue by Karibart** [mraardvark/pyupdi#45]
Hi
Is there a way to write the User Row of the EEPROM with pyupdi?
Or is there a reason why adding this would be a bad idea? I could give it a try.
Thanks!

**Comment by mraardvark** [comment]
Sorry, didn't see this until now.  You can always add it as an optional action.

**Comment by MarkR42** [comment]
Yes there is a way to write the user row, write to fuses >= 128, the user row is 0x80 bytes after the fuses 

So pyupdi.py -fs 128:0x2a sets USERROW0 to 0x2a

**Comment by popovec** [comment]
The USERROW address in DA devices:  0x1080, FUSES:  0x1050 so the function for fuse writing can be used to write into USERROW, but  LOCK at address 0x1040 can not be accessed in this way. Or did I miss something?
